keras_cv/utils/preprocessing.py

```diff
@@ -91,6 +91,23 @@ def blend(image1: tf.Tensor, image2: tf.Tensor, factor: float) -> tf.Tensor:
     temp = image1 + scaled
     return tf.clip_by_value(temp, 0.0, 255.0)
 
+def parse_value_range(param, min_value=0.0, max_value=1.0, param_name="factor"):
```

Comment:
Probably should call it parse_factor_value_range (disambiguous about the input value range)

Reply:
Done